app/src/androidTest/java/com/arygm/quickfix/ui/search/SearchWorkerResultScreenTest.kt

```diff
@@ -212,35 +213,15 @@ class SearchWorkerResultScreenTest {
           searchViewModel,
           accountViewModel,
           userViewModel,
-          preferencesViewModel,
           quickFixViewModel,
+          preferencesViewModel,
           workerViewModel = workerViewModel)
     }
     // Verify that Back and Search icons are present in the top bar
     composeTestRule.onNodeWithContentDescription("Back").assertExists().assertIsDisplayed()
     composeTestRule.onNodeWithContentDescription("Search").assertExists().assertIsDisplayed()
   }
 
-  @Test
-  fun testTitleAndDescriptionAreDisplayed() {
-    // Set the composable content
-    composeTestRule.setContent {
-      SearchWorkerResult(
-          navigationActions,
-          searchViewModel,
-          accountViewModel,
-          userViewModel,
-          preferencesViewModel,
-          quickFixViewModel,
-          workerViewModel = workerViewModel)
-    }
-    // Set the search query and verify that the title and description match the query
-    searchViewModel.setSearchQuery("Unknown")
-
-    // Check if the description with the query text is displayed
-    composeTestRule.onAllNodesWithText("Unknown").assertCountEquals(3)
-  }
-
```

Comment:
Again, why was this test removed ? Once added and passed, tests should in no way be removed. Please restore it

Reply:
This test was removed because the `searchQuery` field in `searchViewModel` appears to be unused and serves no functional purpose in the codebase. As a result, the test doesn’t provide any meaningful validation. At least from my understanding